### PR TITLE
EXT_feature_metadata JSON schema cleanup

### PR DESCRIPTION
**Pull request by kring**
Fixes some small things I noticed while adding this extension to cesium-native:

* Specify that when the min/max/etc properties are arrays they must be arrays of numbers, not arrays of just anything.
* Remove possibly invalid (and redundant anyway) `not.required` property.

### CODE REVIEW

**Comment by kring** [comment]
By the way, are we sure we want specify min/max/etc to be `"type": ["number", "array"]`? glTF accessors have min/max properties that are very similar, and in that case they're always arrays. If the accessor is a scalar, it's simply an array of one element. This is much easier to deal with in statically typed languages.

File: extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/gltf.EXT_feature_metadata.schema.json [hunk 1]
```diff
@@ -35,11 +35,5 @@
     },
     "extensions": {},
     "extras": {}
-  },
-  "not": {
-    "required": [
-      "schema",
-      "schemaUri"
-    ]
```
Comment (lilleyse) [inline extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/gltf.EXT_feature_metadata.schema.json:43]: This is to ensure that `schema` and `schemaUri` are mutually exclusive and was copied from `camera.schema.json` in the glTF spec:

https://github.com/KhronosGroup/glTF/blob/c6a5521ebd1469aba076949a97f150bf3591b63f/specification/2.0/schema/camera.schema.json#L36-L38

I wonder if there's a better way to do this.

File: extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/statistics.class.property.schema.json [hunk 2]
```diff
@@ -61,7 +82,10 @@
         "type": [
           "number",
           "array"
-        ]
+        ],
+        "items": {
+          "type": "number"
+        }
```
Comment (lilleyse) [inline extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/statistics.class.property.schema.json:88]: The type should "integer" in both places here. I just pushed a fix related to that https://github.com/CesiumGS/glTF/commit/b3f48c2a594ac1c6319dd0087b4f5fef2218a832

**Comment by lilleyse** [comment]
> By the way, are we sure we want specify min/max/etc to be `"type": ["number", "array"]`? glTF accessors have min/max properties that are very similar, and in that case they're always arrays. If the accessor is a scalar, it's simply an array of one element. This is much easier to deal with in statically typed languages.

I've debated this too. I'm not a huge fan of single element arrays since scalar properties tend to be way more common in metadata and the array form adds clutter/confusion IMO. I'd rather see the min/max value match the type exactly. Also there is better symmetry with the `"default"` property which doesn't use array form for single elements either.

I think glTF gets away with it because scalar accessors are not all that common. I understand why it's done though.

**Comment by lilleyse** [comment]
@kring thanks for the fixes and sorry for such a late review on this, I missed have must it when you posted it. Looks good I just had a couple comments.

**Comment by lilleyse** [comment]
Merged these changes in https://github.com/CesiumGS/glTF/pull/10

**Pull request closed without merging**